device-types/Cisco/IE-4000-4S8P4G-E.yaml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Possibly missing PSU on this one too.

Reply:
These IE Switches run of external DC and do not have PSUs